src/common/PreviewUtils.ts

```diff
@@ -89,9 +89,9 @@ export class PreviewUtils {
                 errorMessage: errorText,
                 passed: isValid
             });
-        } catch (err: any) {
+        } catch (err) {
             return Promise.resolve({
-                errorMessage: err,
+                errorMessage: (err as any).toString(),
```

Comment:
Just curious. If we left `err` as it is instead of `(err as any).toString()` by virtue of `errorMessage` being a string would we get automatic conversion and not have to do casting to `any` and then getting the string by calling `toString()`?

Reply:
That wouldn't work and ESLint would still throw an error. This is b/c in the newer version of TypeScript, the `error` in a try-catch block is explicitly marked as `unknown` type so it `must` be cast to something before it can be used.